Packs/CommonScripts/ReleaseNotes/1_12_16.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+
+##### ParseEmailFilesV2
+- Updated the Docker image to: *demisto/parse-emails:1.0.0.71712*.
```

Comment:
Add a comment here about the fix this introduces so customers can understand that is solves the given issue.